Hazelnut/src/EditorLayer.cpp

```diff
@@ -179,8 +179,8 @@ namespace Hazel {
 		ImVec2 viewportPanelSize = ImGui::GetContentRegionAvail();
 		m_ViewportSize = { viewportPanelSize.x, viewportPanelSize.y };
 
-		uint32_t textureID = m_Framebuffer->GetColorAttachmentRendererID();
-		ImGui::Image((void*)textureID, ImVec2{ m_ViewportSize.x, m_ViewportSize.y }, ImVec2{ 0, 1 }, ImVec2{ 1, 0 });
+		uint64_t textureID = m_Framebuffer->GetColorAttachmentRendererID();
```

Comment:
`m_Framebuffer->GetColorAttachmentRendererID()` returns a `uint32_t`, shouldn't we at least explicitly cast to `uint64_t`?
https://github.com/TheCherno/Hazel/blob/13b0353ad66172e38767629e36eb1bdb72caf2d0/Hazel/src/Hazel/Renderer/Framebuffer.h#L26

It does look weird on this particular line, I feel we should leave this as a uint32_t and only cast to uint64_t while casting it to a void pointer...

Reply:
I made it there because doing a `reinterpret_cast<void*>(static_cast<uint64_t>(textureID));` looked way like there was way too much going on for just casting to void*. I guess we could also put it on a separate line or `static_cast` the `GetColorAttachmentRendererID()` call, but I thought this was cleaner since there's no reason a 64-bit uint container can't hold a 32-bit uint value in any circumstance.

Reply:
yeah, I guess we can live with it...